pinot-controller/src/main/java/org/apache/pinot/controller/util/TableSizeReader.java

```diff
@@ -248,7 +279,7 @@ public TableSubTypeSizeDetails getTableSubtypeSize(String tableNameWithType, int
       String segment = entry.getKey();
       SegmentSizeDetails sizeDetails = entry.getValue();
       // Iterate over all segment size info, update reported size, track max segment size and number of errored servers
-      long segmentLevelMax = -1L;
+      long segmentLevelMax = DEFAULT_SIZE_WHEN_MISSING_OR_ERROR;
```

Comment:
with `sizeDetails._maxReportedSizePerReplicaInBytes`, we do not need the variable `segmentLevelMax` any more.

Reply:
I agree. 

It's just a minor suggestion to simplify the code a little bit.

Reply:
Sure , I removed the extra variable .